### DISCUSSION TITLE
Importing a file based pool

### DISCUSSION THREAD

**Discussion by EchterAgo** [openzfsonwindows/openzfs#269] [Q&A] [answered]
In the [Windows README](https://github.com/openzfsonwindows/openzfs/tree/windows/module/os/windows) there are instructions on how to set up a file based pool:

```
fsutil file createnew C:\poolfile.bin 200000000
zpool.exe create tank \\?\C:\poolfile.bin
```

After a system restart, how can I import the pool again? Doing `zpool import tank` gives `cannot import 'tank': no such pool available`

**Comment by lundman** [comment, marked as answer]
zpool import -d C:/

ie, pass the directory the pool file(s) are in to -d.
